### PR TITLE
LPS-178475 | Poshi Automation for Bug LPS-167430 | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Poshi Automation for Bug LPS-167430

Hi! :)
This pr is corresponding to this ticket: [LPS-167430](https://liferay.atlassian.net/browse/LPS-167430) → [LPS-178475](https://liferay.atlassian.net/browse/LPS-178475)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase [hunk 1]
```diff
@@ -640,4 +640,37 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-167430. Fragment Menu difficult to Display"
+	@priority = 3
+	test ViewKabobConfigurationMenu {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithSearch.testcase:645]: ViewEllipsisIconWhenHoverOnBodyOfWidget
Move the case to ContentPages.testcase

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, could you simplify cases using the following steps?

1. Add a new site
2. Add a Search Results widget from Commerce category into a content page
3. Hover on the body of widget
4. View the ellipsis icon shown

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 2]
```diff
@@ -1576,6 +1576,39 @@ definition {
...
+		task ("Given a content page with a Search Results widget") {
+			CommerceChannels.openChannelsAdmin();
+
+			Click(locator1 = "Button#PLUS");
+
+			CommerceEntry.addNewChannelsEntry(
+				channelCurrency = "US Dollar",
+				channelName = "Test Channel",
+				channelType = "Site",
+				siteName = "Guest");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1591]: These steps are unnecessary

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 3]
```diff
@@ -1576,6 +1576,39 @@ definition {
...
+				siteName = ${randomSiteName});
+
+			PageEditor.addWidget(portletName = "Search Results");
+		}
+
+		task ("When the mouse passes over the fragment") {
+			MouseOver(
+				index = 1,
+				key_fragmentName = "Search Results",
+				locator1 = "Fragment#FRAGMENT_LABEL");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:1604]: Should be Fragment#FRAGMENT_BODY

**Comment by Tim-Cao** [comment]
@rafaellajordao see changes https://github.com/liferay-echo/liferay-portal/pull/12646

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**